Base/usr/share/man/man1/su.md

```diff
@@ -0,0 +1,41 @@
+## Name
+
+su - run a command with substitute user
```

Comment:
Ehm... that's not what su does. It **s**witches to a **u**ser. The command for running some single thing as another user is `pls`, like `sudo` or `doas` on other *nixes.